df.ui-menus.xml

```diff
@@ -118,11 +118,15 @@
         <int32_t name='dump_x_shift' since='v0.34.08'/>
         <int32_t name='dump_y_shift' since='v0.34.08'/>
         <int32_t name='speed' init-value='50000' since='v0.34.08'/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
-        <int32_t/>
+        <int16_t name='centerx'/>
+        <int16_t name='centery'/>
+        <int32_t name='centerz'/>
```

Comment:
This looks odd to me. Does it still align properly if changed to an int16? If so, the 16 bits following it are probably padding (unused)

Reply:
Also, this could be a `df::coord` instead

Reply:
done